### PR TITLE
MSI: Fix installation steps

### PR DESCRIPTION
**Pull request by rnetser**

### CODE REVIEW

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-operator-v4.13-hypershift-GA-production-smoke-rosa-rhods-hypershift-fips

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-IIB-smoke-rosa-rhods-addon-baseline-fips

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-operator-v4.13-hypershift-GA-production-smoke-rosa-rhods-hypershift-fips

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-IIB-smoke-rosa-rhods-addon-baseline-fips

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-rhods-addon-interop-v4.13-IIB-smoke-rosa-rhods-addon-baseline-fips

File: ci-operator/step-registry/operator/hypershift/operator-hypershift-workflow.yaml [hunk 1]
```diff
@@ -13,13 +13,15 @@ workflow:
...
+      ETCD_ENCRYPTION: "false"
+      FIPS: "false"
     post:
-    - ref: operator-uninstall
     - chain: rosa-aws-sts-deprovision-byo-vpc
     - ref : firewatch-report-issues
     pre:
     - chain: rosa-aws-sts-provision-byo-vpc
-    - ref: osd-ccs-conf-idp-htpasswd-multi-users
+    # - ref: osd-ccs-conf-idp-htpasswd-multi-users
```
Comment (omrirh) [inline ci-operator/step-registry/operator/hypershift/operator-hypershift-workflow.yaml:24]: meant to be removed ?

**Comment by omrirh** [comment]
please fix PR name (isntallation -> installation)

**Comment by rnetser** [comment]
> please fix PR name (isntallation -> installation)

done

File: ci-operator/step-registry/operator/hypershift/operator-hypershift-workflow.yaml (hunk 1 above)
Comment (rnetser) [reply ci-operator/step-registry/operator/hypershift/operator-hypershift-workflow.yaml:24]: for now no; once we have all flows working will remove

**Review by omrirh — APPROVED** [review]

**Pull request merged**